src/plugins/userIndicator/components/MissionStatusPopup.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div class="c-user-control-panel__component">
+    <div class="c-user-control-panel__header">
+      <div class="c-user-control-panel__title">Mission Status</div>
+      <button
+        aria-label="Close"
+        class="c-icon-button c-icon-button--sm t-close-btn icon-x"
+        @click.stop="onDismiss"
+      ></button>
+    </div>
+    <div class="c-ucp-mission-status">
+      <template v-for="status in missionStatuses" :key="status">
```

Comment:
Isn't `status` an object `{key,label}`? Is the key here a placeholder for now?

Reply:
WIP